plugin/markdown/plugin.js

```diff
@@ -174,25 +184,38 @@ const Plugin = () => {
 			wasHorizontal = isHorizontal;
 		}
 
+		if (metadata){
+			try {
+				parsedMetadata = yaml.load(metadata);
+			} catch (error) {
+				console.error("Error while parsing metadata", error)
+			}
+		}
+
 		// add the remaining slide
 		( wasHorizontal ? sectionStack : sectionStack[sectionStack.length-1] ).push( markdown.substring( lastIndex ) );
 
-		var markdownSections = '';
+		let markdownSections = '';
 
 		// flatten the hierarchical stack, and insert <section data-markdown> tags
-		for( var i = 0, len = sectionStack.length; i < len; i++ ) {
+		for( let i = 0, len = sectionStack.length; i < len; i++ ) {
+			let classMetadata = '';
+			if (parsedMetadata) {
+				options.metadata = parsedMetadata.slides[i];
+				classMetadata = ' class=' + options.metadata.slideType;
```

Comment:
if that works
```suggestion
			if (parsedMetadata) {
				options.metadata = parsedMetadata.slides[i];
				options.attributes += ' class=' + options.metadata.slideType;
```

Reply:
Nice catch!!!


:smiley:     ***Reviewing Code, Improves Its Quality***